cob_android_msgs/CMakeLists.txt

```diff
@@ -3,87 +3,11 @@ project(cob_android_msgs)
 
 find_package(catkin REQUIRED COMPONENTS message_generation std_msgs)
```

Comment:
Your srv actually does not depend on std_msgs - so you can remove it from package.xml and CMakeLists